LayoutTests/imported/w3c/web-platform-tests/css/css-fonts/test_font_feature_values_parsing-expected.txt

```diff
@@ -3,47 +3,126 @@ PASS basic parse tests - @font-feature-values;
-FAIL basic parse tests - @font-feature-values bongo { @styleset { abc: 1 2 3 assert_true: unexpected syntax error expected true got false
-FAIL basic parse tests - @font-feature-values bongo { @styleset { ok-1: 1; } } assert_true: unexpected syntax error expected true got false
-FAIL basic parse tests - @font-feature-values bongo { @annotation { ok-1: 3; } } assert_true: unexpected syntax error expected true got false
-FAIL basic parse tests - @font-feature-values bongo { @stylistic { blah: 3; } } assert_true: unexpected syntax error expected true got false
-FAIL basic parse tests - @font-feature-values bongo {
+PASS basic parse tests - @font-feature-values bongo { @blah , @styleset { abc: 1 2 3; } }
+PASS no value definitions in serialization - @font-feature-values bongo { @blah , @styleset { abc: 1 2 3; } }
+PASS basic parse tests - @font-feature-values bongo { @styleset { abc: 1 2 3; }
+PASS serialization check - @font-feature-values bongo { @styleset { abc: 1 2 3; }
+PASS serialization round-trip - @font-feature-values bongo { @styleset { abc: 1 2 3; }
+PASS basic parse tests - @font-feature-values bongo { @styleset { abc: 1 2 3 }
+PASS serialization check - @font-feature-values bongo { @styleset { abc: 1 2 3 }
+PASS serialization round-trip - @font-feature-values bongo { @styleset { abc: 1 2 3 }
+PASS basic parse tests - @font-feature-values bongo { @styleset { abc: 1 2 3;
+PASS serialization check - @font-feature-values bongo { @styleset { abc: 1 2 3;
+PASS serialization round-trip - @font-feature-values bongo { @styleset { abc: 1 2 3;
+PASS basic parse tests - @font-feature-values bongo { @styleset { abc: 1 2 3
+PASS serialization check - @font-feature-values bongo { @styleset { abc: 1 2 3
+PASS serialization round-trip - @font-feature-values bongo { @styleset { abc: 1 2 3
+PASS basic parse tests - @font-feature-values bongo { @styleset { ok-1: 1; } }
+PASS serialization check - @font-feature-values bongo { @styleset { ok-1: 1; } }
+PASS serialization round-trip - @font-feature-values bongo { @styleset { ok-1: 1; } }
+PASS basic parse tests - @font-feature-values bongo { @annotation { ok-1: 3; } }
+PASS serialization check - @font-feature-values bongo { @annotation { ok-1: 3; } }
+PASS serialization round-trip - @font-feature-values bongo { @annotation { ok-1: 3; } }
+PASS basic parse tests - @font-feature-values bongo { @stylistic { blah: 3; } }
+PASS serialization check - @font-feature-values bongo { @stylistic { blah: 3; } }
+PASS serialization round-trip - @font-feature-values bongo { @stylistic { blah: 3; } }
+PASS basic parse tests - @font-feature-values bongo {
 @styleset
   { blah: 3; super-blah: 4 5;
   more-blah: 5 6 7;
- } } assert_true: unexpected syntax error expected true got false
-FAIL basic parse tests - @font-feature-values bongo {
+ } }
+FAIL serialization check - @font-feature-values bongo {
```

Comment:
FAIL sounds bad

Reply:
We don't keep the order of the descriptors in this patch, which is the reason of the failure.
Maybe we should ? Or maybe we should modify the test ?